ephios/plugins/complexsignup/templates/complexsignup/vue_editor.html

```diff
@@ -0,0 +1,374 @@
+                                                </span>
+                                            </span>
+                                            <span v-else>
+                                                {% translate "unnamed" %}
+                                            </span>
+                                        </span>
+                                        <span v-for="sub_block in getSubBlocks(getBlockById(composition.sub_block))"
+                                              :key="sub_block.uuid"
+                                              class="badge text-bg-secondary me-1">
+                                            <span v-if="sub_block.name">
+                                                [[ sub_block.name ]]
+                                            </span>
+                                            <span v-else>
+                                                {% translate "unnamed" %}
+                                            </span>
+                                        </span>
+                                    </small>
+                                </div>
+                                <div class="col-auto">
+                                    <div class="form-check">
+                                        <input type="checkbox" v-model="composition.optional"
+                                               class="form-check-input"
+                                               :id="'composition-' + composition.clientId"
+                                        />
+                                        <label class="form-check-label" :for="'composition-' + composition.clientId">
+                                            {% translate "optional" %}
+                                        </label>
+                                    </div>
+                                </div>
+                                <div class="col-auto">
+                                    <button type="button" class="btn btn-sm btn-danger"
+                                            @click="removeSubComposition(currentBlock, composition)">
+                                        <i class="fas fa-trash"></i>
+                                        <span class="visually-hidden">
+                                            {% translate "Remove" %}
+                                        </span>
+                                    </button>
+                                </div>
+                            </div>
+                        </li>
+                        <li class="list-group-item list-group-item-secondary text-center">
+                            {% translate "Add items using the buttons in the list" %}
+                        </li>
+                    </ul>
+                    <div v-if="currentBlock.block_type === 'atomic'" class="mb-3 form-check d-inline-block">
+                        <input type="checkbox"
```

Comment:
does not get saved

Reply:
good catch! Apparently I never tested that. Had the variable name in the frontend all wrong (`allowMore` instead of `allow_more`)